src/virtual-machine.js

```diff
@@ -129,7 +128,7 @@ class VirtualMachine extends EventEmitter {
             this.emit(Runtime.BLOCK_UPDATE, blockId, blockInfo);
         });
         this.runtime.on(Runtime.TOOLBOX_EXTENSIONS_NEED_UPDATE, () => {
-            this.extensionManager.refreshBlocks();
+            this.runtime.extensionManager.refreshBlocks();
```

Comment:
I think there's no longer to bubble this event all the way up to the VM anymore. We could either remove the event all together or just have the runtime listen and handle its own event since it also owns the extensionManager.